docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md

```diff
@@ -2,7 +2,7 @@
 
 The [api:SS_List] class is designed to return a flat list of records.
 These lists can get quite long, and hard to present on a single list.
-[Pagination](/templates/how_tos/pagination) is one way to solve this problem,
+[Pagination](/en/3.0/howto/pagination/) is one way to solve this problem,
```

Comment:
I'm pretty sure the https://github.com/silverstripe/silverstripe-docsviewer module adds the prefix part of the URL on. I haven't see this fully qualified type of URL in any other doc file before.

@wilr any ideas about why there are no pagination docs in the new set up?